main/webapp/modules/core/externals/suggest/suggest-4_3a.js

```diff
@@ -724,11 +727,7 @@
       var css = this.options.css;
       var li = $("<li>").addClass(css.item);
       var label = $("<label>").text(data.name);
-      var div = $("<div>").addClass(css.item_name).append(label);
-      if(data.description) {
-          div.append($('<span></span>').text(data.description));
-      }
-      li.append(div);
```

Comment:
This removes support for descriptions in suggested items.

Reply:
Isn't the description rendered in the preview pane (aka flyout)? Does this change cause it to show up twice?

Reply:
This is a description that is rendered in suggest items by default, without needing to hover the item. You are correct that it is redundant with the flyout service - I have introduced it a while ago as a replacement for flyouts because it gives a better user experience (having to hover one item at a time to find the right one is not really workable). Services can decide not to return descriptions and use flyouts, or the converse. (The Wikidata service currently uses both, but the flyouts should just be removed https://github.com/wetneb/openrefine-wikibase/issues/77)

Reply:
I don't think I agree, but my intent was not remove local changes. It turns out that there were only a couple of changes in SVN, one of which was the one needed for jQuery compatibility, so I just integrated those by diffing against the base version before we applied our local changes.